### PR TITLE
fix(tusd): workaround image url suffix

### LINKED ISSUE #1140: fix(profile): picture does not load
The profile picture currently does not load, but it should.

### PR DESCRIPTION
**Pull request by dargmuesli**
Resolves #1140

Unsure why image loading stopped working.
We currently only save the `uploadId` and not the multipart `uploadId`. Both ids are separated by a plus sign. Adding a character after the plus sign, like another one, seems to fill in the multipart `uploadId`, preventing an error. Again, I don't know why leaving the second id out ever worked.
We now use both ids.

### CODE REVIEW

**Comment by PatrickMi** [comment]
I tried out the PR. When going to the images page I still saw an error when an image is loaded. Also when trying to upload an image I can select an image from my computer but nothing happens afterwards. No Modal to select the crop is opened.

**Review by PatrickMi — APPROVED** [review]
Works for me now!

EDIT: Still does not work. I have to check, how I can take approvals back

EDIT2: Looks like I can't dismiss reviews somehow

**Comment by PatrickMi** [comment]
I'm sorry, I made a mistake. I checked the wrong feature. Unfortunately the image upload is still not working for me. I can open can image from my computer, select a crop in the modal, click on upload but I get an error then. In the gallery I then see the red error image telling me that an error happened.

**Comment by dargmuesli** [comment]
There is no such thing as **an** error for devs, there is only **the** error X :wink: cuz a defined error is an error that can be fixed way better than *something*.
Everything works for me. The error message would help, maybe check the running services for issues or create a `Cypress` test that shows the error on CI :bulb:

**Comment by dargmuesli** [comment]
Also see https://github.com/maevsi/maevsi/issues/1165, maybe the image file is just too large.

**Comment by PatrickMi** [comment]
> There is no such thing as **an** error for devs, there is only **the** error X 😉 cuz a defined error is an error that can be fixed way better than _something_.
> Everything works for me. The error message would help, maybe check the running services for issues or create a `Cypress` test that shows the error on CI 💡

You are right. I rushed here a bit. I tried it again with an image that has a size of 12kb and got the same error. The console has the following error:
`Access to XMLHttpRequest at 'https://tusd.localhost/files/' from origin 'https://localhost' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.` 
This seems to be a problem with my local config. Maybe we can check what the problem here is?
Bildschirm­foto 2023-04-04 um 14 30 38

**Pull request merged**
